ui/src/telegrafs/components/Collectors.tsx

```diff
@@ -286,7 +257,7 @@ class Collectors extends PureComponent<Props, State> {
 }
 const mstp = ({telegrafs, orgs: {org}, buckets}: AppState): StateProps => {
   return {
-    collectors: telegrafs.list,
+    hasTelegrafs: !!telegrafs.list.length,
```

Comment:
i think the more javascript idiomatic way to do this is

```ts
hasTelegrafs: telegrafs.list && telegrafs.list.length > 0
```

i like this personally because it lets the reader understand the intent with fewer mental hops:

```ts
telegrafs.list && telegrafs.list.length > 0
```
the telegrafs has a list - > the list has membership

_vs_

```ts
!!telegrafs.list.length
```
not not telegrafs.list.length -> boolean cast and truthy evaluation -> figure out truthiness of `list.length` (`list.length === 0 // false`, `list.length !== 0 // true`) -> list has membership

Reply:
i wonder if we standardize on one way or the other. it seems like the `!!` is the 'future' https://github.com/influxdata/influxdb/pull/16296/files/0a881156f9839487a1b1431d88e86c2899202ca4#diff-583598f3738558469237fb30beb2e1bdR77

thoughts??